src/react/account/AccountHead.jsx

```diff
@@ -34,17 +30,13 @@ function AccountHead({ account }: Props) {
         dispatch(closeAccountDeleteDialog());
     };
 
-    // TODO: Should we let the user delete accounts that still owns buckets.
     return (
         <Head>
             { !!account && <DeleteConfirmation show={showDelete} cancel={handleDeleteCancel} approve={handleDeleteApprove} titleText={`Are you sure you want to delete account: ${account.userName} ?`}/> }
```

Comment:
Can you move `DeleteConfirmation` component to `/details/Properties.js` page. You won't need to check if the account exists. Here is an exemple: https://github.com/scality/zenko-ui/blob/development/1.0/src/react/databrowser/buckets/details/Overview.jsx#L83

Reply:
done